main.py

```diff
@@ -11,12 +16,12 @@ def main():
     try:
         connection_parameters = parse_connection_arguments()
     except RuntimeError as ex:
-        print(f'[Err] exception has been caught during argument parsing: {ex}')
+        logger.critical(f'[Err] exception has been caught during argument parsing: {ex}')
```

Comment:
Same comments as for necessity of [Err] and exc_info=True parameter.

Reply:
resolved